### PR TITLE
Improve deploy-tag

### PR DESCRIPTION
**Pull request by szepeviktor**
https://docs.github.com/en/rest/issues/milestones?apiVersion=2022-11-28#update-a-milestone

### CODE REVIEW

File: .github/workflows/reusable-deploy-tag.yml [hunk 1]
```diff
@@ -110,8 +110,6 @@ jobs:
         git config user.email github-actions@github.com
         git add ${{ inputs.readme }}
         git commit -m "Readme"
-      env:
-        TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (szepeviktor) [inline .github/workflows/reusable-deploy-tag.yml:114]: No push here. `git commit` goes only to the local repo.

File: .github/workflows/reusable-deploy-tag.yml [hunk 2]
```diff
@@ -161,7 +159,7 @@ jobs:
 
     - name: Close completed milestone
       run: | #shell
-        ID=$(gh api "/repos/${{ github.repository }}/milestones" | jq -r '.[] | select(.title == "${{ inputs.version }}") | .number')
-        gh api "/repos/${{ github.repository }}/milestones/$ID" --field "state=closed"
+        ID=$(gh api "/repos/${{ github.repository }}/milestones" --jq '.[] | select(."title" == "${{ inputs.version }}") | ."number"')
+        gh api "/repos/${{ github.repository }}/milestones/${ID}" --method "PATCH" --field "state=closed"
```
Comment (szepeviktor) [inline .github/workflows/reusable-deploy-tag.yml:163]: For `--jq` please see `gh api --help`.

**Comment by szepeviktor** [comment]
Glad to contribute.

**Comment by johnbillion** [comment]
I wish I could get nektos/act running locally but I just get endless errors.

**Pull request merged**